docs/guides/development/pack-maker-tools.md

```diff
@@ -0,0 +1,108 @@
+
+=== "Doc menu"
+    Click the three dots icon to the right of the doc's name and select **Toggle Pack maker tools**.
+
+    <img src="../../../images/pmt_open_doc.png" srcset="../../../images/pmt_open_doc_2x.png 2x" class="screenshot" alt="Open Pack maker tools from doc menu">
+
+=== "Pack side panel"
+    Open the side panel for a Pack ({{ coda.pack_panel_clicks }}) and click the button **Open Pack maker tools**. This option is only visible for users with edit access to the Pack.
+
+    <img src="../../../images/pmt_open_panel.png" srcset="../../../images/pmt_open_panel_2x.png 2x" class="screenshot" alt="Open Pack maker tools from Pack panel">
+
+=== "Error message"
+    Hover over the broken Pack formula and click **View error details**. This option is only visible for users with edit access to the Pack.
+
+    <img src="../../../images/pmt_open_error.png" srcset="../../../images/pmt_open_error_2x.png 2x" class="screenshot" alt="Open Pack maker tools from a formula error">
+
+The Pack maker tools opens in the document as a panel at the bottom of the document, that you can resize or close.
+
+<img src="../../../images/pmt_overview.png" srcset="../../../images/pmt_overview_2x.png 2x" class="screenshot" alt="The Pack maker tools">
+
+
+## Selecting the Pack
+
+The Pack maker tools only works with one Pack at a time. To change the selected Pack you can click the Pack name at the top of the panel and select a new one from the dropdown. You'll only be able to select Packs that are installed in the document and that you have edit access to.
+
+<img src="../../../images/pmt_select.png" srcset="../../../images/pmt_select_2x.png 2x" class="screenshot" alt="Change the selected Pack">
+
+
+## Logs
+
+The primary feature of the Pack maker tools is the ability to look at the logs for your Pack. The Pack maker tools will only show the log entries for the selected Pack within the current doc. If you need to see the log entries for your Pack within another doc, you'll first need to get access to that document.
+
+New logs entries are displayed automatically, usually a few seconds after the Pack is run. There is no need to refresh the page or open and close and Pack maker tools. Logs are kept for approximately two weeks, after which they are no longer available.
+
+
+### Executions
+
+The logs for your Pack are grouped by execution, in reverse chronological order (latest executions at the top).
+
+<img src="../../../images/pmt_overview.png" srcset="../../../images/pmt_overview_2x.png 2x" class="screenshot" alt="Execution logs">
+
+A new execution is logged each time your Pack is run, for example when a formula is calculated or a button is pressed. When a sync table syncs you'll see an execution for the initial sync, and additional execution for each continuation of the sync. Additionally, while a user is working in the formula editor your Pack's formula may be run multiple times in order to generate a preview.
```

Comment:
Might add some more detail about duplicate executions, along the lines of

"You may see duplicate executions for the same formula. This could be due to multiple browser clients using the doc the at the same time, Coda's servers regularly snapshotting the doc, or to using the formula editor, which invokes formulas as you type to generate previews of the output."

Reply:
Good idea, done. PTAL.